server/tsconfig.json

```diff
@@ -4,10 +4,10 @@
     "allowJs": true,
     "skipLibCheck": true,
     "strict": true,
-    "noEmit": true,
+    "noEmit":false,
     "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "bundler",
+    "module": "commonjs",
```

Comment:
This might impact other imports in the application. Please run the application and see if everything is working as expected.

Reply:
Sir, I made the corrections and run the application again. Everything works as expected. please let me know if there's anything else to check.